buildvertical.targets

```diff
@@ -8,6 +8,22 @@
  <!-- Required for FindBestConfiguration task -->
  <Import Project="$(MSBuildThisFileDirectory)src/Tools/GenerateProps/properties.props" />
 
+ <Target Name="AnnotateProjects"
+         BeforeTargets="BuildAllProjects"
+         Condition="!$(MSBuildProjectFullPath.EndsWith('.proj'))">
```

Comment:
If the `MSBuildProjectFullPath` is a '.builds' file, then the Projects are the csproj / vbproj files (probably), so we want to annotate those projects.  We don't necessarily need to skip this for 'builds.proj', but we can.